src/test/dogecoin_tests.cpp

```diff
@@ -12,6 +12,108 @@
 
 BOOST_FIXTURE_TEST_SUITE(dogecoin_tests, TestingSetup)
 
+/**
+ * the maximum block reward at a given height for a block without fees
+ */
+uint64_t expectedMaxSubsidy(int height) {
```

Comment:
where is this (and `expectedMinSubsidy`) used?

Reply:
They're not, it turns out, it's a mistake copied from 1.14. Removing.